src/web/waiters/OperationsWaiter.mjs

```diff
@@ -167,18 +183,43 @@ class OperationsWaiter {
      * @param {event} e
      */
     opListCreate(e) {
+        if (window.innerWidth < this.app.breakpoint) {
+            this.mobileOpListCreate(e);
+        } else {
+            this.desktopOpListCreate(e);
+        }
+    }
+
+    /**
+     * Create the desktop op-list which allows popovers
+     * and dragging
+     *
+     * @param {event} e
+     */
+    desktopOpListCreate(e) {
         this.manager.recipe.createSortableSeedList(e.target);
-        this.enableOpsListPopovers(e.target);
+        this.enableOpPopover(e.target);
+    }
+
+    /**
+     * Create the mobile op-list which does not allow
+     * popovers and dragging
+     *
+     * @param {event} e
+     */
+    mobileOpListCreate(e) {
+        this.manager.recipe.createSortableSeedList(e.target, false);
+        this.disableOpsListPopovers();
     }
 
 
     /**
-     * Sets up popovers, allowing the popover itself to gain focus which enables scrolling
-     * and other interactions.
+     * Enable the target operation popover itself to gain focus which
+     * enables scrolling and other interactions.
      *
      * @param {Element} el - The element to start selecting from
      */
-    enableOpsListPopovers(el) {
+    enableOpPopover(el) {
         $(el).find("[data-toggle=popover]").addBack("[data-toggle=popover]")
```

Comment:
I imagine you are required to use jQuery here in order to be able to chain the `.popover()` method? If not, try to remove jQuery reliance.

Reply:
Yes, this also related to the .tooltip("enable") / .tooltip("disable") you mentioned above in setMobileUI and setDesktopUI. It doesn't work with querySelectorAll. I imagine this will have the same result. Put a pin in it and circle back to it later ;)